Lesson22/Merve/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>To-Do App</title>
+		<link rel="stylesheet" type="text/css" href="styles.css" />
+	</head>
+	<body>
+        
+		<div class="container">
+			<h1>To-Do List</h1>
+			<form id="toDoForm">
+				<p id="errorMsg"></p>
```

Comment:
Place this <p> element outside the form element, better below it otherwise layout breaks when an error message is displayed. Would be good to also add styles to this element to make it visually easy to understand that it is caused by an error.